taca/utils/statusdb.py

```diff
@@ -83,57 +83,61 @@ def get_project_flowcell(
 
 
 class ProjectSummaryConnection(StatusdbSession):
-    def __init__(self, config, dbname="projects"):
+    def __init__(self, config):
         super().__init__(config)
-        self.db = self.connection[dbname]
+        self.projects_db = self.connection["projects"]
         self.name_view = {
-            k.key: k.id for k in self.db.view("project/project_name", reduce=False)
+            k.key: k.id
+            for k in self.projects_db.view("project/project_name", reduce=False)
         }
         self.id_view = {
-            k.key: k.id for k in self.db.view("project/project_id", reduce=False)
+            k.key: k.id
+            for k in self.projects_db.view("project/project_id", reduce=False)
         }
 
 
 class FlowcellRunMetricsConnection(StatusdbSession):
-    def __init__(self, config, dbname="flowcells"):
+    def __init__(self, config):
         super().__init__(config)
-        self.db = self.connection[dbname]
+        self.flowcells_db = self.connection["flowcells"]
         self.name_view = {k.key: k.id for k in self.db.view("names/name", reduce=False)}
         self.proj_list = {
             k.key: k.value
-            for k in self.db.view("names/project_ids_list", reduce=False)
+            for k in self.flowcells_db.view("names/project_ids_list", reduce=False)
             if k.key
         }
 
 
 class X_FlowcellRunMetricsConnection(StatusdbSession):
     def __init__(self, config, dbname="x_flowcells"):
```

Comment:
Is the dbname here used in taca_ngi_pipeline?

Reply:
Yes you're right. It is reused in the common methods in the StatusDBSession class. Hmm, might have to rethink this.